### ISSUE TITLE
Dimension mismatch error in MAGEMin_wrappers.jl

### ISSUE THREAD

**Issue by lcandiot** [ComputationalThermodynamics/MAGEMin#80]
Dear all,

I noticed recently that Julia throws a `dimension mismatch error` when I give a custom bulk rock input in mol-% without specifying H2O as a major oxide. The error occurs in MAGEMin_wrappers.jl in line 469 and an example that should reproduce the error is given below. If I am not mistaken, H2O content is set to 0 implicitly when the bulk rock is given in wt-%. In that case the error does not occur. Has anyone encountered the same issue or am I making a mistake somewhere? 

```
    data  = Initialize_MAGEMin("ig", verbose=false);
    n       = 10
    P       = fill(10.0,n)
    T       = fill(1100.0,n)
    Xoxides = ["SiO2"; "Al2O3"; "CaO"; "MgO"; "FeO"; "Fe2O3"; "K2O"; "Na2O"; "TiO2"; "Cr2O3"];
    X       = [48.43; 15.19; 11.57; 10.13; 6.65; 1.64; 0.59; 1.87; 0.68; 0.0];
    sys_in  = "mol"
    out     = multi_point_minimization(P, T, data, X=X, Xoxides=Xoxides, sys_in=sys_in)
    Finalize_MAGEMin(data)
```

Best,
Lorenzo

**Comment by NicolasRiel** [comment]
Hi Lorenzo,

Yes, this will be fixed in the next release. For now it is safer to provide all oxides of the database.

Thanks for reporting the problem!

Nicolas

**Comment by NicolasRiel** [comment]
The issue has been fixed in version 1.3.9 I now close it
